azure/notebooks/kubernetes/Dask_Kube_XGBoost.ipynb

```diff
@@ -0,0 +1,917 @@
+{
```

Comment:
<p>Can we rename the notebook to MNMG_XGBoost?</p><p> </p><p>*LocalCUDACluster since we're extending single node multi-GPU to multi-node</p><p> </p><p>*leverage multiple nodes and GPUs using </p><p> </p><p>*Forest Inference Library (FIL)</p><p> </p><p>*we will use part of the</p><p> </p><p>*Azure Open Datasets (can you also link tit)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Actually you can do both since dask-kubernetes is agnostic, and you can choose not to use GPU as well. I thought something like "<a href="https://kubernetes.dask.org/en/latest/" target="_blank">Dask Kubernetes</a>&nbsp;is a library that extends a single machine multi-CPU&nbsp;<a href="https://docs.dask.org/en/latest/setup/single-distributed.html#localcluster" target="_blank">LocalCluster</a>&nbsp;or a single machine multi-GPU&nbsp;<a href="https://docs.rapids.ai/api/dask-cuda/nightly/api.html#cluster" target="_blank">LocalCUDACluster</a>&nbsp;for use in workloads that can leverage multiple nodes and GPUs using Kubernetes." would be useful. </p><p> </p><p>What do  you think?</p>